app/assets/stylesheets/styles/components/guides/components/_actions_modal.css.scss

```diff
@@ -0,0 +1,33 @@
+.actions-modal {
+  -webkit-transition: opacity 200ms ease-in;
```

Comment:
Properties should be ordered -moz-transition, -webkit-transition, transition<br>Avoid vendor prefixes.